Dockerfile

```diff
@@ -8,10 +8,10 @@ LABEL Description="This image can be used to create a sidekick container for rec
 LABEL maintainer="Richard North <rich.north@gmail.com>"
 
 RUN apt-get update && apt-get install -y \
-  python-pip python-dev ffmpeg=7:4.1.6-1~deb10u1 \
+  python-pip python-dev ffmpeg \
```

Comment:
Ah, yeah, I should have mentioned why. :)

The Dockerfile as it existed didn't build anymore on either amd64 or arm64 because that pinned version isn't available in the repos used:
```
$ docker build --platform amd64 . 2>&1 | grep "'ffmpeg' was not found"
#5 21.42 E: Version '7:4.1.6-1~deb10u1' for 'ffmpeg' was not found
$ docker build --platform arm64 . 2>&1 | grep "'ffmpeg' was not found"
#5 5.159 E: Version '7:4.1.6-1~deb10u1' for 'ffmpeg' was not found
```

Dropping the pinned version resolved that (and it builds on all five platforms now), and it seems to work fine for me on arm64 and amd64.

The version that gets installed for me on arm64 is `4.1.9-0+deb10u1`.